assets/src/components/d3/createResourceAccessChart.js

```diff
@@ -170,6 +177,7 @@ function createResourceAccessChart ({ data, width, height, domElement }) {
 
     bar.enter()
       .append('rect')
+      .attr('tabindex', d => d.index * 10 + 1)
```

Comment:
Interesting idea. Did you try to just set this to 0 for all of the elements? 0 should just navigate in sequence.

Reply:
I really can't remember if I did or not. I know without this it would simply tab down the file names.  Based on the way the elements were laid out I think it was going to tab through all the file names sequentially then all the bars, but I'm not sure if  I observed that or simply suspected it...